### PR TITLE
Minor Fixes to get migrations to work on CI

### PR DESCRIPTION
**Pull request by daltonfury42**
Minor changes to make the migrations to work on the backend server.

### CODE REVIEW

File: simplq/src/main/resources/db/changeset.log.xml
Comment (daltonfury42) [inline simplq/src/main/resources/db/changeset.log.xml:73]: @chalx We need a way to catch schema mismatches premerge. Is this possible?

What I have in mind is to run a postgres docker instance, apply migrations, and run our current integration tests against it.

Comment (chalx) [reply simplq/src/main/resources/db/changeset.log.xml:73]: @daltonfury42 yes we can do this. This is what I also have in mind. I will look on this.

Comment (chalx) [reply simplq/src/main/resources/db/changeset.log.xml:73]: @daltonfury42 added in this pr #121

Comment (daltonfury42) [reply simplq/src/main/resources/db/changeset.log.xml:73]: I've set a branch protection rule that says I will need one approval for my PR to be merged. 

This way, you would also get an idea about all changes that go it. Can you approve PRs once they look ok?

Comment (daltonfury42) [reply simplq/src/main/resources/db/changeset.log.xml:73]: Also let me know if we need to change this. This system is working well for the react repo.

**Review by chalx — APPROVED** [review]
Everything looks fine

**Pull request merged**
